apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html

```diff
@@ -0,0 +1,205 @@
+<form class="content" [formGroup]="visualizationForm" (ngSubmit)="onSubmit()" (keydown.enter)="$event.preventDefault()">
+  <span class="title">Create a Cell Distance Visualization</span>
+
+  <mat-card class="creation-step data">
+    <mat-card-header>
+      <mat-card-title>
+        <div class="step-number">1</div>
+        Format and Upload Data
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-actions>
+      <div class="left-side">
+        <div class="column-content">
+          <h2 class="subtitle">Upload Cell Type Table</h2>
+          <div class="column-info">
+            <div class="column-info-column">
+              <span class="column-info-header">Required Columns</span>
+              <span class="column-info-list">X, Y, and Cell Type</span>
+            </div>
+            <div class="column-info-column">
+              <span class="column-info-header">Optional Columns</span>
+              <span class="column-info-list">Z and Cell Ontology ID</span>
+            </div>
+          </div>
+          @if (dataUploaded) {
+            <div class="column-info-column">
+              <span class="column-info-header">File loaded:</span>
+              <span class="column-info-list">(file name)</span>
+            </div>
+            <button mat-button (click)="removeCSV()">Remove CSV</button>
+          } @else {
+            <button mat-button (click)="upload()">Upload CSV</button>
```

Comment:
I think it might be a good idea to turn this into a file-select component. The more I read about creating a file upload button I discovered it will need quite a few additions to work nicely and still be accessible